hack/istio/install-fraud-detection-demo.sh

```diff
@@ -62,6 +62,7 @@ fi
 echo "IS_OPENSHIFT=${IS_OPENSHIFT}"
 echo "IS_MAISTRA=${IS_MAISTRA}"
 
+# If we are to delete, remove everything and exit immediately after
 if [ "${DELETE_DEMO}" == "true" ]; then
```

Comment:
is it me or are you missing the `fi` to close this if-stmt?

Reply:
that's a comment

Reply:
I dunno, maybe you need to refresh